### PR TITLE
yank: add page

### PR DESCRIPTION
**Pull request by monsieurlo**
tldr page for the yank (https://github.com/mptre/yank) utility.

----

- [x] The page (if new), does not already exist in the repo.

- [x] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited.

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines.

### CODE REVIEW

File: pages/common/yank.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# yank
```
Comment (pxgamer) [inline pages/common/yank.md]: Do you know if Yank works on Windows? I can't see any install instructions in the README.

Comment (monsieurlo) [reply pages/common/yank.md]: AFAIK it doesn't...

Comment (agnivade) [reply pages/common/yank.md]: Then you should duplicate the page separately in linux and osx folders.

File: pages/common/yank.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+- Yank a field from a CSV file:
+
+`yank -d \", <file.csv`
+
+- Yank a whole line using the -l option:
+
+`make 2>&1 | yank -l`
+
+- If stdout is not a terminal the selected field will be written to stdout and exit without invoking the yank command. Kill the selected PID:
```
Comment (agnivade) [inline pages/common/yank.md]: This description is a bit too long. Can we shorten this ?

File: pages/common/yank.md (hunk 1 above)
Comment (monsieurlo) [reply pages/common/yank.md]: Makes sense ^^
Does it need to create another pull request?

File: pages/common/yank.md (hunk 2 above)
Comment (monsieurlo) [reply pages/common/yank.md]: It comes from the documentation of the tool and I don't really see how to shorten it. But we can delete this specific example if you think it's not ok.

File: pages/common/yank.md (hunk 1 above)
Comment (agnivade) [reply pages/common/yank.md]: You can just update this pull request itself.

File: pages/common/yank.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
+# yank
+
+> Read input from stdin and display a selection interface that allows a field to be selected and copied to the clipboard.
+
+- Yank an environment variable key or value:
+
+`env | yank -d =`
```
Comment (agnivade) [inline pages/common/yank.md]: Change to - `{{echo hello=world}} | yank -d {{=}}`

File: pages/common/yank.md (hunk 2 above)
Comment (agnivade) [reply pages/common/yank.md]: I see that you have just lifted the descriptions verbatim from the documentation. We usually explain the descriptions in a language simple enough for beginners to understand. How about - `Select a field and pipe to stdout without copying to clipboard` ?

File: pages/common/yank.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
+# yank
+
+> Read input from stdin and display a selection interface that allows a field to be selected and copied to the clipboard.
+
+- Yank an environment variable key or value:
```
Comment (agnivade) [inline pages/common/yank.md]: Start with the simplest command invocation - `yank`. Explain what it does and give an example.

File: pages/common/yank.md [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+
+`env | yank -d =`
+
+- Yank a field from a CSV file:
```
Comment (agnivade) [inline pages/common/yank.md]: We can remove this example. Usually we have one example per flag.

File: pages/common/yank.md [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+
+`yank -d \", <file.csv`
+
+- Yank a whole line using the -l option:
```
Comment (agnivade) [inline pages/common/yank.md]: Again, this is verbatim from the docs. I don't think there is a need to mention 'using the -l' option. It is clear from the example that -l is used. I would just say - `Yank an entire line to clipboard:`

File: pages/common/yank.md [hunk 7]
```diff
@@ -0,0 +1,23 @@
...
+
+`make 2>&1 | yank -l`
```
Comment (agnivade) [inline pages/common/yank.md]: Change this to - `echo -e "hello\nworld" | yank -l`

Comment (agnivade) [reply pages/common/yank.md]: I chose `echo` because it's a simple command and shows the functionality clearly that it is printing multiple lines. But if you think `dmesg` is better, that is also fine.

**Comment by agnivade** [comment]
Please move the file from common and duplicate across linux and osx folders.
